schema.json

```diff
@@ -5,7 +5,7 @@
   "type": "object",
   "properties": {
     "references": {
-      "default": ["master"],
```

Comment:
Probably keep both to avoid breaking all the preexisting repos on GitHub

Reply:
dione broo